drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -1088,9 +1088,9 @@ def process_record(record:, hud_values:, user:, save: true)
       )
       form_processor = process_record(record: client, hud_values: hud_values, user: hmis_user, save: false)
 
-      expect(client.valid?).to eq(false)
+      expect(client.valid?(:client_form)).to eq(false)
```

Comment:
Do you want to check here that client.valid? is true?  (I think that would confirm that a non-context client is still considered valid.